addons/point_of_sale/models/res_config_settings.py

```diff
@@ -104,6 +104,8 @@ def _default_pos_config(self):
     pos_warehouse_id = fields.Many2one(related='pos_config_id.warehouse_id', readonly=False, string="Warehouse (PoS)")
     point_of_sale_use_ticket_qr_code = fields.Boolean(related='company_id.point_of_sale_use_ticket_qr_code', readonly=False)
     pos_auto_validate_terminal_payment = fields.Boolean(related='pos_config_id.auto_validate_terminal_payment', readonly=False, string="Automatically validates orders paid with a payment terminal.")
+    pos_share_orders = fields.Boolean(related='pos_config_id.share_orders', readonly=False)
```

Comment:
need an onchange when we set it to False => should remove all the trusted config